packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -138,8 +140,26 @@ export const _RemoteVideoTile = React.memo(
       );
     }, [renderElement, showLoadingIndicator]);
 
+    /* @conditional-compile-remove(pinned-participants) */
+    const onKeyDown = useCallback(
+      (e: KeyboardEvent) => {
+        if (e.key === 'Enter') {
+          setDrawerMenuItemProps(
+            convertContextualMenuItemsToDrawerMenuItemProps(contextualMenuProps, () => setDrawerMenuItemProps([]))
+          );
+        }
+      },
+      [setDrawerMenuItemProps, contextualMenuProps]
+    );
+
     return (
-      <Stack style={remoteVideoTileWrapperStyle}>
+      <Stack
+        /* @conditional-compile-remove(pinned-participants) */
+        tabIndex={remoteVideoTileMenuKind === 'drawer' ? 0 : undefined}
+        /* @conditional-compile-remove(pinned-participants) */
+        onKeyDown={remoteVideoTileMenuKind === 'drawer' ? onKeyDown : undefined}
```

Comment:
should this handler only be used when its a drawer over a context menu? the handler you wrote will work for desktop no?

Reply:
Yes, the handler should only be assigned when menuKind = 'drawer'. The RemoteVideoTile will be assigned menuKind = 'drawer' when the VGL is on mobile and menuKind = 'contextual' when the VGL is on desktop. We don't want the handler to be active on Desktop like so:
https://user-images.githubusercontent.com/79475487/212398985-b2a781c6-be38-4525-ba1a-ff46c9faae11.mp4